project-base/docs/introduction/custom-entities.md

```diff
@@ -0,0 +1,43 @@
+# Custom Entities
+
+The system is prepared and configured for custom entities.
+If we keep following instructions, entities work without any further effort.
+
+* Place entities in the namespace `Shopsys\ShopBundle\Model` (directory `src/Shopsys/ShopsysBundle/Model`).
```

Comment:
* We decided that we don't want to tell the reader about configuration, what is default, how to change it during PIA. Because it forces us to explain much more - what is the configuration, why do we use annotations, how to change directory, how to change mapping type, keep the configuration link up-to-date.
* The directive style is on purpose. Follow instructions and You are safe. "should" raises questions - "What if I don't want?", and then we have to explain